src/Umbraco.Core/Persistence/DbConnectionExtensions.cs

```diff
@@ -77,6 +77,8 @@ public static bool IsAvailable(this IDbConnection connection)
             }
             catch (DbException)
             {
+                // Don't swallow this error, the exception is super handy for knowing "why" its not available
+                LogHelper.Info<DatabaseContext>("Configured database is reporting as not being available!", DbException );
```

Comment:
Perhaps you should log this as an WarnWithException, and then pass in the exception... you also need to define the exception variable... this will not build ;)